browser/net/brave_profile_network_delegate.cc

```diff
@@ -49,6 +50,9 @@ BraveProfileNetworkDelegate::BraveProfileNetworkDelegate(
   callback = base::Bind(brave::OnBeforeURLRequest_TorWork);
   before_url_request_callbacks_.push_back(callback);
 
+  callback = base::Bind(brave::OnBeforeURLRequest_TranslateRedirectWork);
```

Comment:
We're only supposed to use `BindOnce` or `BindRepeating` now, because `Bind` is going away soon (I got that feedback in a couple of reviews so far, just passing it along).